test/bench_test.go

```diff
@@ -306,24 +306,22 @@ func benchOptionsForServiceImports() *server.Options {
 func addServiceImports(b *testing.B, s *server.Server) {
 	// Add a bunch of service exports with wildcards, similar to JS.
 	var exports = []string{
-		server.JetStreamEnabled,
```

Comment:
we had plans to have this one always respond even when jetstream isnt enabled, will we likewise use account info for that?

Reply:
Yes that is correct. That should be the behavior. I was toying with the idea of just having them all respond all the time regardless. WDYT?

Reply:
Will leave as is for now. Some more thought needs to go on this one wrt advisories etc.